compose_generator/odcs_fetcher.py

```diff
@@ -20,5 +25,35 @@ class ODCSFetcher(ComposeFetcher):
     Fetch ODCS compose based on a remote compose-reference and store it locally
     """
 
+    compose_path: Optional[Path] = None
```

Comment:
Let's document this one as well. Mainly need to clarify whether this should include the file name or just the containing folder structure.

Reply:
I think we can make this input mandatory and assume it's provided.
The reason being that I don't see how we can return the output path to the user, so it should be provided to us.
In that case, we can assume the path provided includes the path together with the filename.

Reply:
Changed, but it does seem less user-friendly.

Reply:
I think the user here is going to be a dockerfile that we would write, so I don't expect that to matter.